Cargo.toml

```diff
@@ -83,9 +83,10 @@ wasm-bindgen = ["rand/wasm-bindgen"]
 # since rust 1.26.0
 u128 = ["byteorder"]
 
+# since rust 1.31.0
+const_fn = []
```

Comment:
why do we need to keep the feature? since it will be a part of all versions we support and i don't see a reason for any user to opt-in or opt-out of it.

Reply:
since currently people may be using `uuid` transitively and dont want their code to break. `0.8.0` will remove them (should prolly document that)

Reply:
Ah that sounds fair. At least before `1.0` we should review and deprecate any features that we don't need anymore. But we have removed unstable features from `lazy-static` in minor versions post `1.0` so shouldn't feel like we have to be stuck with them forever in any case.

Reply:
well with `0.8.0` it will happen anyhow as we will transition to `2018` edition which will automatically guarantee that `const_fn` feature is available. 1.31.0 introduced `const_fn` and `2018` edition is `1.32.0`